beacon_node/network/src/sync/block_lookups/mod.rs

```diff
@@ -739,6 +752,8 @@ impl<T: BeaconChainTypes> BlockLookups<T> {
             "Block component processed for lookup";
             "response_type" => ?R::response_type(),
             "block_root" => ?root,
+            "result" => ?result,
```

Comment:
I think this might result in logging the entire `SignedBeaconBlock`?
https://github.com/sigp/lighthouse/blob/21cdc64bfe75572170c61b28ed5a972f36b354ac/beacon_node/beacon_chain/src/block_verification.rs#L146-L153

Reply:
I think it's ok to leave it as it is - from what I've seen the logged `result` seems pretty helpful.

Reply:
just saw one and it's quite huge 😅 I'm going to add a manual `Debug` impl fro `RpcBlock`, because I think the `result` logging is quite nice and probably worth keeping.

Reply:
https://github.com/sigp/lighthouse/pull/5488/commits/476f49d695f676ece9d2aa95deacd3a93bf2185e